how-to-use-azureml/ml-frameworks/pytorch/distributed-pytorch-with-distributeddataparallel/distributed-pytorch-with-distributeddataparallel.ipynb

```diff
@@ -293,7 +293,7 @@
       "source": [
         "from azureml.core import Environment\n",
         "\n",
-        "pytorch_env = Environment.get(ws, name='AzureML-PyTorch-1.6-GPU')"
+        "pytorch_env = Environment.get(ws, name='azureml-acpt-pytorch-1.11-cuda11.3')"
```

Comment:
please change to azureml-acpt-pytorch-2.0-cuda11.7
torch 1.11 is being deprecated.

Reply:
Thanks.   I'll try updating that in a separate PR.